themes/src/main/resources/theme/base/admin/resources/js/controllers/clients.js

```diff
@@ -1314,6 +1314,12 @@ module.controller('ClientDetailCtrl', function($scope, realm, client, flows, $ro
         var attrVal5 = $scope.client.attributes['ciba.backchannel.auth.request.signing.alg'];
         $scope.cibaBackchannelAuthRequestSigningAlg = attrVal5==null ? 'none' : attrVal5;
 
+        var attrVal6 = $scope.client.attributes['request.object.encryption.alg'];
+        $scope.requestObjectEncryptionAlg = attrVal6==null ? 'any' : attrVal6;
+
+        var attrVal7 = $scope.client.attributes['request.object.encryption.enc.alg'];
+        $scope.requestObjectEncryptionEncAlg = attrVal7==null ? 'any' : attrVal7;
```

Comment:
If https://github.com/keycloak/keycloak/pull/8261/files#r672633565 is agreed, it might be better to be `'request.object.encryption.enc.alg'` .

Reply:
Also to be `requestObjectEncryptionEnc`